charts/brigade/templates/_helpers.tpl

```diff
@@ -16,7 +16,7 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- end -}}
 
 {{- define "brigade.gw.fullname" -}}
-{{ include "brigade.fullname" . | printf "%s-gw" }}
+{{ include "brigade.fullname" . | printf "%s-github-gw" }}
```

Comment:
I had considered to name it `gh-gw` after the existing `cr-gw` for consistency but refrained from doing so.
Perhaps in the future, we are likely to add `bitbucket-gw`, `gitlab-gw` and so on which doesn't have a well-known abbrev form? Then, a name like `gh-gw` will lose consistency very soon.

Reply:
I agree with your decision.